Assets/PlayerHealth.cs

```diff
@@ -27,7 +27,4 @@ void Start()
     {
         int currentHealth = maxHealth;  //ilość zdrowia gdy nie ponieśliśmy żadnych obrażeń
     }
-
-}
-
-
+}
```

Comment:
missing newline at the end of the file :D. There should be no unnecessary new lines in our files but git likes it that the files end with one new line :) so i suggest adding 1 new line.